timesketch/frontend/src/components/Sketch/TimeFilter.vue

```diff
@@ -130,6 +134,19 @@ export default {
       this.$emit('addChip', this.chip)
       this.startDateTime = ''
       this.endDateTime = ''
+
+      // Close the menu
+      this.$parent.$parent.isActive = false;
```

Comment:
This is difficult to reason about. What prop is being mutated here? In what component. Can you emit an event and catch that in the calling component instead?